es-core/src/Settings.cpp

```diff
@@ -225,6 +225,19 @@ void Settings::loadFile()
 		setFloat(node.attribute("name").as_string(), node.attribute("value").as_float());
 	for(pugi::xml_node node = doc.child("string"); node; node = node.next_sibling("string"))
 		setString(node.attribute("name").as_string(), node.attribute("value").as_string());
+
+	processBackwardCompatiblity();
+}
+
+void Settings::processBackwardCompatiblity()
```

Comment:
I like this idea in general, and would be a proponent of such methods. That being said, since this is a one-shot operation for most people, I'd be happy with removing this as well - just a thought.

Reply:
-- this is not an objection, but more to get the perspective from others on whether it's worth (and encouraged) to start adding this.